pkg/oc/cli/cmd/rsync/copyrsync.go

```diff
@@ -111,6 +110,21 @@ func (r *rsyncStrategy) Validate() error {
 	return nil
 }
 
+// rsyncEscapeCommand wraps each element of the command in double quotes
+// if it contains any of the following: single quote, double quote, backslash, or space
+// It also replaces every pre-existing double quote character in the element with a pair.
+// Example: " -> ""
+func rsyncEscapeCommand(command []string) []string {
```

Comment:
note that this function modifies its argument in place and returns it, which is a little unusual